x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_charts_container.js

```diff
@@ -34,8 +34,8 @@ import { addItemToRecentlyAccessed } from '../../util/recently_accessed';
 
 const textTooManyBuckets = i18n.translate('xpack.ml.explorer.charts.tooManyBucketsDescription', {
   defaultMessage:
-    'This selection contains too many buckets to be displayed.' +
-    'The dashboard is best viewed over a shorter time range.',
+    'This selection contains too many buckets to be displayed. ' +
```

Comment:
Nit: this line break could be done with template literals.

Reply:
Removed line break in this warning message in https://github.com/elastic/kibana/pull/81291/commits/f34db019428380856663a2640d4c48102f09dcd8. The original change was done to add a missing whitespace char between the two sentences in this message. There doesn't need to be actual line break in the tooltip message.